grails-app/init/de/iteratec/osm/FrontendWatcher.groovy

```diff
@@ -25,15 +25,26 @@ class FrontendWatcher {
     protected void startFrontendWatcher(Environment environment) {
 
         String location = environment.getReloadLocation()
-        def nodeLocation = new File(new FileNameByRegexFinder().getFileNames(location, "/build/nodejs/([^\\\\/]*)/bin/node")[0]).getParent()
+
+        def nodeExecLocation = new FileNameByRegexFinder().getFileNames( Paths.get(location, "build", "nodejs").toString(), "[\\\\/]([^\\\\/]*)[\\\\/]((node\\.exe+)|(bin/node))")[0]
```

Comment:
This may not find the right executable on Linux anymore, since all subsequent bin/node expressions will also be matched. Try following expression:

`new FileNameByRegexFinder().getFileNames( Paths.get(location, "build", "nodejs").toString(), "([^\\/]*)((node\\.exe+)|(/bin/node(?!.)))")[0]`

Please  verify that this also works in Windows with the inverted slashes.